libs/editor/src/main/java/org/wordpress/android/editor/gutenberg/GutenbergKitEditorFragment.java

```diff
@@ -129,6 +130,11 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle sa
             mSettings = (Map<String, Object>) getArguments().getSerializable(ARG_GUTENBERG_KIT_SETTINGS);
         }
 
+        // request dependency injection. Do this after setting min/max dimensions
+        if (getActivity() instanceof EditorFragmentActivity) {
+            ((EditorFragmentActivity) getActivity()).initializeEditorFragment();
+        }
```

Comment:
This pattern mirrors that found in the Aztec and Gutenberg Mobile editor implementations. 

https://github.com/wordpress-mobile/WordPress-Android/blob/25ac30b8d55169d36b5b936df6f9e6870b551b47/libs/editor/src/main/java/org/wordpress/android/editor/AztecEditorFragment.java#L258-L261

https://github.com/wordpress-mobile/WordPress-Android/blob/25ac30b8d55169d36b5b936df6f9e6870b551b47/libs/editor/src/main/java/org/wordpress/android/editor/gutenberg/GutenbergEditorFragment.java#L568-L571